docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -277,6 +222,8 @@ const PostType = () => {
 export default connect(PostType);
 ````
 
+---
+
 ## More things added
 
 ### `actions.source.updateState`
```

Comment:
I don't think we need the following sections in the docs:
- Out of Scope
- API changes
- Breaking changes
- Deprecated APIs